src/main/java/UserInterface.java

```diff
@@ -0,0 +1,223 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class UserInterface {
+    public static final String COMMAND_EXIT = "bye";
+    public static final String COMMAND_LIST = "list";
+    public static final String COMMAND_MARK = "mark";
+    public static final String COMMAND_UNMARK = "unmark";
+    public static final String COMMAND_TODO = "todo";
+    public static final String COMMAND_DEADLINE = "deadline";
+    public static final String COMMAND_EVENT = "event";
+    private final Scanner uiScan;
```

Comment:
Constants name should be all capital letters with the words being separated by an underscore.

Reply:
Thank you. I will change this.